cmd/podman/import.go

```diff
@@ -7,10 +7,11 @@ import (
 	"net/http"
 	"net/url"
 	"os"
-	"strings"
 
 	"github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
+	cliutil "github.com/projectatomic/libpod/cmd/podman/util"
+	rt "github.com/projectatomic/libpod/cmd/podman/util"
```

Comment:
Huh why are you including the same thing twice with two different names?

Reply:
Better question: Why does Go even allow that without compile errors

Reply:
fixed, just an oversight